clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/security/CloudFoundryCredentials.java

```diff
@@ -49,7 +50,7 @@
 
   public CloudFoundryCredentials(String name, String appsManagerUri, String metricsUri, String apiHost, String userName, String password, String environment) {
```

Comment:
Can you mark environment as `@Nullable`?

Reply:
👍